client/lcd/swagger-ui/swagger.yaml

```diff
@@ -522,190 +605,930 @@ paths:
+      - in: query
+        name: simulate
+        description: if true, ignore the gas field and perform a simulation of a transaction, but don't broadcast it
+        required: false
+        type: boolean
+      - in: query
+        name: generate_only
+        description: if true, build an unsigned transaction and write it back
+        required: false
+        type: boolean
+      - type: string
+        description: proposal id
+        name: proposalId
+        required: true
+        in: path
+      - description: valid value of `"option"` field can be `"yes"`, `"no"`, `"no_with_veto"` and `"abstain"`
+        name: post_vote_body
+        in: body
+        required: true
+        schema:
+          type: object
+          properties:
+            base_req:
+              "$ref": "#/definitions/BaseReq"
+            voter:
+              "$ref": "#/definitions/Address"
+            option:
+              type: string
+      responses:
+        200:
+          description: OK
+          schema:
+            "$ref": "#/definitions/BroadcastTxCommitResult"
+        400:
+          description: Invalid proposal id or vote body
+        401:
+          description: Key password is wrong
+        500:
+          description: Internal Server Error
+    get:
+      summary: Query voters
+      description: Query voters information by proposalId
+      produces:
+      - application/json
+      tags:
+      - ICS22
+      parameters:
+      - type: string
+        description: proposal id
+        name: proposalId
+        required: true
+        in: path
+      responses:
+        200:
+          description: OK
+          schema:
+            type: array
+            items:
+              "$ref": "#/definitions/Vote"
+        204:
+          description: No content
+        400:
+          description: Invalid proposal id
+        500:
+          description: Internal Server Error
+  /gov/proposals/{proposalId}:
+    get:
+      summary: Query a proposal
+      description: Query a proposal by id
+      produces:
+      - application/json
+      tags:
+      - ICS22
+      parameters:
+      - type: string
+        name: proposalId
+        required: true
+        in: path
+      responses:
+        200:
+          description: OK
+          schema:
+            "$ref": "#/definitions/TextProposal"
+        204:
+          description: No content
+        400:
+          description: Invalid proposal id
+        500:
+          description: Internal Server Error
+  /gov/proposal/{proposalId}/deposits:
+    get:
+      summary: Query deposits
+      description: Query deposits by proposalId
+      produces:
+      - application/json
+      tags:
+      - ICS22
+      parameters:
+      - type: string
+        name: proposalId
+        required: true
+        in: path
+      responses:
+        200:
+          description: OK
+          schema:
+            type: array
+            items:
+              "$ref": "#/definitions/TextProposal"
```

Comment:
This should be `Deposits` and should go next to the `POST .../deposits` request

Reply:
```suggestion
              "$ref": "#/definitions/Deposit"
```

Reply:
I'm sorry for my mistake. I will recheck my change again.